docs/UserGuide.md

```diff
@@ -136,7 +136,13 @@ UI mockup:
 ---
 ### `addi` : Adding an interview to an applicant
 
-Adds a new interview to an applicant.
+Adds a new interview to an applicant. In the case of duplicate names, the system will automatically increment the last 
+number in the user input by 1, or add 1 if there is no number.
+
+Duplicate handling: 
+* Entering `technical12`, then `technical12` again will result in the 2nd entry being converted to 
+`technical13`.
+* Entering `technical`, then `technical` will result in the 2nd entry being converted to `technical1`.
```

Comment:
Optional: you may want to include again after `technical` for consistency with the previous line, but it's a minor issue